js/src/components/google-account-card/switch-account-button.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import AppButton from '.~/components/app-button';
+import useSwitchGoogleAccount from './useSwitchGoogleAccount';
+
+/**
+ * Clicking on the "connect to a different Google account" button.
+ *
+ * @event gla_google_account_connect_different_account_button_click
+ */
+
+/**
+ * Renders a switch button that lets user connect with another Google account.
+ *
+ * @fires gla_google_account_connect_different_account_button_click
+ * @param {Object} props React props
+ * @param {string} [props.text="Or, connect to a different Google account"] Text to display on the button
+ */
+const SwitchAccountButton = ( {
+	text = __(
+		'Or, connect to a different Google account',
+		'google-listings-and-ads'
+	),
+	...restProps
+} ) => {
+	const [ handleSwitch, { loading } ] = useSwitchGoogleAccount();
```

Comment:
It was not introduced by this PR but the `useSwitchGoogleAccount` hook needs to disconnect Google Ads account as well. Maybe the [`disconnectAllAccounts` action](https://github.com/woocommerce/google-listings-and-ads/blob/ba22a30b21d62392c95855eeaed1aa2c812a4cac/js/src/data/actions.js#L511-L538) could do it.

Reply:
Good catch! I opted to use an approach consistent to how the MC account is currently disconnected in [3b9dbbb](https://github.com/woocommerce/google-listings-and-ads/pull/2660/commits/3b9dbbbf19d60ab999d7c5d59ca91d589724def2). I believe the `disconnectAllAccounts` action will also cause the WP.org account to be disconnected, which we don't want.